pilot/pkg/keycertbundle/watcher.go

```diff
@@ -113,7 +108,6 @@ func (w *Watcher) SetFromFilesAndNotify(keyFile, certFile, rootCert string) erro
 	if len(caBundle) != 0 {
 		w.bundle.CABundle = caBundle
 	}
-	w.initDone.Store(true)
```

Comment:
Can you please call `SetAndNotify` after file is read so that this logic is unified?

Reply:
sounds good